src/js/components/FileInput/doc.js

```diff
@@ -0,0 +1,159 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils/mixins';
+import { themeDocUtils } from '../../utils/themeDocUtils';
+
+export const doc = FileInput => {
+  const DocumentedFileInput = describe(FileInput)
+    .availableAt(getAvailableAtBadge('FileInput'))
+    .description('A control to input one or more files.')
+    .usage(
+      `import { FileInput } from 'grommet';
+<FileInput id='file' name='file' />`,
+    )
+    .intrinsicElement('input');
+
+  DocumentedFileInput.propTypes = {
+    accept: PropTypes.string.description(
+      'MIME type pattern to match against. For example: "image/*".',
+    ),
```

Comment:
Since we are not modifying this prop in any way and its a plainly HTML attribute, why do you think we should document it?

Reply:
Well, another reviewer asked about how to do it. I can remove it. But, it would be good to come up with a way to document this sort of thing better. Perhaps we should have a link in the documentation to the MDN page for the intrinsic element?

Reply:
Agreed. https://github.com/grommet/grommet/issues/4969